generate_workspace/src/main/java/com/google/devtools/build/workspace/output/BzlWriter.java

```diff
@@ -41,7 +43,9 @@ public BzlWriter(String[] argv, String outputDirStr) {
 
   @Override
   public void write(Collection<Rule> rules) {
-    try (PrintStream outputStream = new PrintStream(generatedFile.toFile())) {
+    try {
```

Comment:
Move the createParentDirectory out of the try/catch and continue to use try-with-resources.